src/components/tile/tile.component.tsx

```diff
@@ -33,13 +33,17 @@ export interface TileProps extends SpaceProps, WidthProps {
     | "negative"
     | "caution"
     | "info";
+  /** Sets the level of roundedness of the corners, "low" (default) is 8px and "high" is 16px */
```

Comment:
Again as previously mentioned stick with one spelling and I think it's not clear with the variable names not matching the description.